drakcore/drakcore/app.py

```diff
@@ -134,6 +134,13 @@ def graph(task_uid):
         return send_file(f.name, mimetype='text/plain')
 
 
+@app.route("/sha256/<task_uid>")
+def sha256(task_uid):
+    with NamedTemporaryFile() as f:
+        minio.fget_object("drakrun", f"{task_uid}/sample_sha256.txt", f.name)
+        return send_file(f.name, mimetype='text/plain')
```

Comment:
AFAIK `sample_sha256.txt` is considered "legacy". I would expose `metadata.json` via API and present the user interface with the ability to query the whole JSON, not just a single field

Reply:
Good point. I've added /metadata/<task_uid> endpoint.